build/pom.xml

```
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>cloud-openstack</artifactId>
         <groupId>jetbrains.buildServer.clouds</groupId>
-        <version>1.3-SNAPSHOT</version>
+        <version>1.3</version>
```

Development version (-SNAPSHOT) should be kept, release will done on dedicated process.

```suggestion
        <version>1.3-SNAPSHOT</version>
```